src/Settings/BankingInformationSettings/BankingInformationSettingsForm.js

```diff
@@ -0,0 +1,74 @@
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+
+import {
+  Button,
+  Checkbox,
+  Col,
+  Pane,
+  PaneFooter,
+  PaneHeader,
+  Row,
+} from '@folio/stripes/components';
+import stripesForm from '@folio/stripes/final-form';
+
+const BankingInformationSettingsForm = ({
+  handleSubmit,
+  pristine,
+  submitting,
+}) => {
+  const paneFooter = useMemo(() => {
+    const end = (
+      <Button
+        id="clickable-save-contact-person-footer"
+        type="submit"
+        buttonStyle="primary mega"
+        disabled={pristine || submitting}
+        onClick={handleSubmit}
+      >
+        <FormattedMessage id="ui-organizations.settings.accountTypes.save.button" />
+      </Button>
+    );
+
+    return <PaneFooter renderEnd={end} />;
+  }, [handleSubmit, pristine, submitting]);
+
+  const paneTitle = <FormattedMessage id="ui-organizations.settings.bankingInformation" />;
+
+  return (
+    <Pane
+      defaultWidth="fill"
+      id="banking-information"
+      renderHeader={renderProps => <PaneHeader {...renderProps} paneTitle={paneTitle} />}
+      footer={paneFooter}
+    >
+      <Row>
+        <Col xs={12}>
+          <Field
+            component={Checkbox}
+            label={<FormattedMessage id="ui-organizations.settings.bankingInformation" />}
+            name="value"
+            type="checkbox"
+            vertical
```

Comment:
I think to be consistent we should keep label and checkbox horizontally:
https://folio-snapshot.dev.folio.org/settings/orders/approvals
https://folio-snapshot.dev.folio.org/settings/orders/open-order
https://folio-snapshot.dev.folio.org/settings/orders/instance-matching
https://folio-snapshot.dev.folio.org/settings/invoice/approvals
...